build.gradle

```diff
@@ -309,7 +309,17 @@ dependencies {
     compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: versions.launchDarklySdk
     implementation "io.github.openfeign:feign-httpclient:10.7.0"
     implementation 'org.springframework.boot:spring-boot-starter-validation'
-
+    implementation('org.apache.httpcomponents:httpclient:4.5.13') {
+      because 'Apache HttpClient versions prior to version 4.5.13 can misinterpret malformed authority component in ' +
+        'request URIs passed to the library as java.net.URI object and pick the wrong target host for request execution.'
+    }
+    implementation('org.hibernate:hibernate-core:5.4.24.Final') {
+      because 'A flaw was found in hibernate-core in versions prior to and including 5.4.23.Final. ' +
+        'A SQL injection in the implementation of the JPA Criteria API can permit unsanitized literals when a ' +
+        'literal is used in the SQL comments of the query. This flaw could allow an attacker to access unauthorized ' +
+        'information or possibly conduct further attacks. The highest threat from this vulnerability is to data ' +
+        'confidentiality and integrity.'
+    }
     compile group: 'org.flywaydb', name: 'flyway-core', version: '5.2.4'
```

Comment:
@SujeethKumarAri  - can you check the dependency tree and see if the old versions of these jars are included or not?

Reply:
I ran the below commands 

- `./gradlew dependencyInsight --dependency org.hibernate:hibernate-core` and it produced the below result

`Task :dependencyInsight
org.hibernate:hibernate-core:5.4.24.Final
   variant "compile" [
      org.gradle.status              = release (not requested)
      org.gradle.usage               = java-api
      org.gradle.libraryelements     = jar (compatible with: classes)
      org.gradle.category            = library

      Requested attributes not found in the selected variant:
         org.gradle.dependency.bundling = external
         org.gradle.jvm.version         = 11
   ]
   Selection reasons:
      - Was requested : A flaw was found in hibernate-core in versions prior to and including 5.4.23.Final. A SQL injection in the implementation of the JPA Criteria API can permit unsanitized literals when a literal is used in the SQL comments of the query. This flaw could allow an attacker to access unauthorized information or possibly conduct further attacks. The highest threat from this vulnerability is to data confidentiality and integrity.
      - Selected by rule

org.hibernate:hibernate-core:5.4.24.Final
\--- compileClasspath

org.hibernate:hibernate-core:5.4.21.Final -> 5.4.24.Final
\--- org.springframework.boot:spring-boot-starter-data-jpa:2.3.4.RELEASE
     \--- compileClasspath`


- `./gradlew dependencyInsight --dependency org.apache.httpcomponents:httpclient` produced the result
` Task :dependencyInsight
org.apache.httpcomponents:httpclient:4.5.13
   variant "compile" [
      org.gradle.status              = release (not requested)
      org.gradle.usage               = java-api
      org.gradle.libraryelements     = jar (compatible with: classes)
      org.gradle.category            = library

      Requested attributes not found in the selected variant:
         org.gradle.dependency.bundling = external
         org.gradle.jvm.version         = 11
   ]
   Selection reasons:
      - Selected by rule
      - Was requested : Apache HttpClient versions prior to version 4.5.13 can misinterpret malformed authority component in request URIs passed to the library as java.net.URI object and pick the wrong target host for request execution.

org.apache.httpcomponents:httpclient:4.5.13
\--- compileClasspath

org.apache.httpcomponents:httpclient:4.5.3 -> 4.5.13
+--- com.warrenstrange:googleauth:1.1.5
|    \--- uk.gov.hmcts.reform:service-auth-provider-client:3.1.1
|         \--- compileClasspath
\--- io.rest-assured:rest-assured:3.3.0
     \--- compileClasspath

org.apache.httpcomponents:httpclient:4.5.10 -> 4.5.13
\--- io.github.openfeign:feign-httpclient:10.7.0
     +--- compileClasspath
     \--- uk.gov.hmcts.reform:idam-client:1.3.1 (requested io.github.openfeign:feign-httpclient:10.2.3)
          \--- compileClasspath

org.apache.httpcomponents:httpclient:4.5.12 -> 4.5.13
\--- org.apache.httpcomponents:httpmime:4.5.12
     \--- io.rest-assured:rest-assured:3.3.0 (requested org.apache.httpcomponents:httpmime:4.5.3)
          \--- compileClasspath

(*) - dependencies omitted (listed previously)`